<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QMainWindow" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gomoku Online</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../res/icon.png</normaloff>../../res/icon.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableBoard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>402</width>
      <height>402</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>14</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>15</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>16</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>17</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>18</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>19</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="listPlayer">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelListPlayer">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player List:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRoom">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Room:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textRoom">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>#HMIFDrama</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlay">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>151</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start Game!</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listChat">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelChat">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chat Box</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textChat">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Say hello :)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonChat">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>339</y>
      <width>32</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
